components/s3/setup.ftl

```diff
@@ -108,6 +109,7 @@
 
         [@createBlobServiceContainer 
             name=containerId
+            parentNames=[accountId, blobId]
```

Comment:
Are the same parent names required fro every blobServiceContainer? 

if they are it might be cleaner to make these dedicated attributes which you smush together inside the resource, storageAccountId and storageBlobId,. with parentNames it kinda implies you could give it any parents.

Reply:
@roleyfoley  I have addressed this in the [latest commit](https://github.com/codeontap/gen3-azure/pull/40/commits/2a995e8da099bfb69948f4a5f273f244add89c67).